### ISSUE TITLE
CrossEntropyLoss & TripletLoss issue

### ISSUE THREAD

**Issue by JIaqiYang78** [microsoft/Relation-Aware-Global-Attention-Networks#24]
When I trained this model, I have set up Lr=0.0008, weight_decay=0.0005 and I have checked the dim of  feat_ and cls_feat, all of them are correct. But the CrossEntropyLoss fluctuates around 6 and it didn't reduce with epochs increasing. And the TripletLoss is 0.00000 or 0.45000 and in most cases it is 0.0000. And the prec is very low which fluctuates between 0 and 1. 
Does anyone know the reason for this porblem? May be we can discuss about it. Thanks.
